entgql/internal/todo/ent/schema/todo.go

```diff
@@ -29,6 +29,8 @@ type Todo struct {
 	ent.Schema
 }
 
+type Map map[string]any
```

Comment:
This type is not needed IMO:

```go
field.JSON("init", map[string]any{}).
	Optional().
	Annotations(entgql.Type("Map")),
```

Reply:
<img width="1418" alt="Screen Shot 2022-10-06 at 13 40 26" src="https://user-images.githubusercontent.com/12751435/194232105-6d369e56-8a85-4c83-a838-bcf374eea11f.png">
gqlgen generate wrong type if we don't create a new type for the map.